config/config.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/tkanos/gonfig"
+)
+
+type BotConfig struct {
+	CoinAPIURL            string
+	WizdaddyURL           string
+	InvalidCommandMessage string
+}
+
+var (
+	config BotConfig
+)
+
+func GetConfig() *BotConfig {
+	// Initalize config if it doesn't already exist
+	if config == (BotConfig{}) {
```

Comment:
does this needs () ? why can't you just use BotConfig{} to get zero-value struct